github/ci/prow/files/jobs/kubevirt/kubevirt-csi/kubevirt-csi-postsubmits.yaml

```diff
@@ -0,0 +1,52 @@
+postsubmits:
+  kubevirt/kubevirt-csi:
+    - name: publish-kubevirt-csi
+      branches:
+        - master
+      always_run: true
```

Comment:
I think this one is superfluous on postsubmits.

Reply:
done